### PR TITLE
Disable deployer status notifications and re-enable phantom tests.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3873/
Test PASSed.

File: jujugui/static/gui/src/app/views/bundle-import-notifications.js [hunk 1]
```diff
@@ -38,6 +38,13 @@ YUI.add('bundle-import-notifications', function(Y) {
     watchAll: function(env, db) {
       env.deployerStatus(function(response) {
         if (response.err) {
+          // This is expected in the cases the GUI is not served by the GUI
+          // server, in which case the deployer method is not implemented.
+          if (response.err.indexOf('not implemented') > -1) {
+            console.log(
```
Comment (jaycee) [inline jujugui/static/gui/src/app/views/bundle-import-notifications.js]: A neat thing either @hatched or @huwshimi showed me--you can do console.error for something like this as well, which browsers handle slightly different (e.g. firefox dumps a little more info about where the error came from).

**Comment by jaycee** [comment]
:+1: looks good, thanks @frankban.

**Comment by jaycee** [comment]
QA:

In HJC I used a package for jujugui generated from this tarball. On opening the gui I saw no error notification.

QA OK.

**Comment by frankban** [comment]
Thank you JC for the review and QA!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**